homeassistant/components/hangouts/.translations/de.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
This won't work. You need to define a `components/hangouts/strings.json` with the English translations, during development you can run `script/translations_develop` to generate the `en.json` file. All other files are downloaded automatically from our translation platform.